packages/elements-core/src/components/Docs/HttpOperation/Parameters.tsx

```diff
@@ -91,7 +91,7 @@ const httpOperationParamsToSchema = ({ parameters, parameterType }: ParametersPr
     const paramStyle = paramStyleUnspecified
       ? undefined
       : style && defaultStyle[parameterType] !== style
-      ? readableStyles[style] || style
+      ? readableStyles[style as keyof {}] || style
```

Comment:
this doesn't make much sense? what is it trying to do?

Reply:
good question. I'm not 100% sure what this code is trying to achieve since I was mostly concerned about removing type issues. I tried to improve readability and type coverage as best as I could in ae0bba3b1165b8c83f0d19b1cae550426e19ef16

Reply:
theres more usages of it - can we apply the same philosophy?

Reply:
Sounds good. I did my best in ba9c5b1b854fb5856909c2788e56f861958f3bec: I removed all `as keyof {}` and added other improvements. Type coverage should look better now. Let me know if there is anything else that you want to fix.